src/bitcoin-cli.cpp

```diff
@@ -857,8 +857,7 @@ static UniValue ConnectAndCallRPC(BaseRequestHandler* rh, const std::string& str
             }
             break; // Connection succeeded, no need to retry.
         } catch (const CConnectionFailed& e) {
-            const auto now{GetTime<std::chrono::microseconds>()};
-            if (fWait && (timeout <= 0 || now < deadline)) {
+            if (fWait && (timeout <= 0 || GetTime<std::chrono::microseconds>() < deadline)) {
```

Comment:
wouldn't it be better to use `std::chrono::steady_clock` while touching this line?